app/assets/stylesheets/regions/mastheads/boxes-masthead.scss

```diff
@@ -227,47 +227,4 @@
     min-height: 340px !important;
     padding-bottom: 0;
   }
-}
-
```

Comment:
Well yes the remaining code in this file (boxes-masthead.scss) is for boxes. Some of the code that was originally in the masthead.scss was moved to slider-masthead-old.scss though.